flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/client/HiveShimV100.java

```diff
@@ -364,9 +366,29 @@ public LocalDateTime toFlinkTimestamp(Object hiveTimestamp) {
 		return ((Timestamp) hiveTimestamp).toLocalDateTime();
 	}
 
+	@Override
+	public Object toHiveDate(Object flinkDate) {
+		ensureSupportedFlinkDate(flinkDate);
+		return flinkDate instanceof Date ? flinkDate : Date.valueOf((LocalDate) flinkDate);
+	}
+
+	@Override
+	public LocalDate toFlinkDate(Object hiveDate) {
+		Preconditions.checkArgument(hiveDate instanceof Date,
+				"Expecting Hive timestamp to be an instance of %s, but actually got %s",
```

Comment:
c&p typo: timestamp => Date

Reply:
nice catch :)

Reply:
I will check in a hotfix to fix this typo